### ISSUE TITLE
"Amount" output column should have commas stripped out

### ISSUE THREAD

**Issue by lyallcooper** [aniav/ynab-csv#34]
I've been running into some issues where YNAB is incorrectly interpreting the amount field seemingly because the "amount" contains a comma and no trailing cents.

For example, `"$1,150"` is imported as `$1.15`. I believe the tool could solve this by normalizing the output to either have no commas, or always have a trailing `.00` if it's a whole-dollar amount. However, given that some currencies use `,` as the decimal separator and some have no fractional amounts, it might be best to leave it as an option.

**Comment by aniav** [comment]
Thank you for the report. At some point I decided to not interpret the values coming from banks at all, as this was causing far more issues. I'd like to avoid that as much as possible and leave parsing to YNAB.

Did you try changing the budget settings in YNAB to check if it would help with imports?
<img width="469" alt="Screenshot 2020-06-16 at 20 13 22" src="https://user-images.githubusercontent.com/182546/84812066-1d8c8300-b00e-11ea-886f-1a64a74a9247.png">

**Comment by lyallcooper** [comment]
Yeah I agree that the best solution would be for YNAB to fix it on their end (or for my source to always include trailing `.xx`), parsing is generally a bit of a nightmare 😅 

I have my number format set to `123,456.78`, which seems like the right option for my case.

Interestingly it seems YNAB uses slightly different logic in their CSV parsing vs. UI input, as if I input `$1,150` through the UI it parses it to `$1,150.00` as expected.

**Comment by aniav** [comment]
Well, that has to be annoying. I guess when you add stuff through UI it's doing some magic in frontend before it's sending to backend, where the CSV import is probably happening only in backend.

I'll keep this issue as opened and I'll see if I can add anything next time I get to improving the app, I'm not doing a lot of maintenance these days here 🙌

**Comment by bermlee** [comment]
I've run into this issue, and have raised this with YNAB support, as I work with Korean KRW (but still using YNAB), which currency amounts don't have a decimal point (not since massive inflation in the 20th century).

Could there be some logic on this end such as see that if there are three trailing digits at the end, then assume that the comma is not a fractional part of a decimal number and should be stripped out? 

Or at the least a checkbox option "assume whole numbers only"?

**Comment by aniav** [comment]
@bermlee dod you get any response from YNAB support around this?
